python/packages/ai/teams/ai/planner/openai_planner_options.py

```diff
@@ -11,30 +11,9 @@
 class OpenAIPlannerOptions:
     api_key: str
     default_model: str
-    prompt_folder: str
+    prompt_folder: str = "prompts"
     organization: Optional[str] = None
     endpoint: Optional[str] = None
     one_say_per_turn: bool = False
     use_system_message: bool = False
     log_requests: bool = False
```

Comment:
The constructor is added on purpose to declare some parameter is keyword only, please do not remove it

Reply:
dataclasses should not have constructors because they provide there own default constructor for the class. The way to initialize fields of a dataclass is via the method above using property initialization.

Reply:
ok thanks for explaining that to me in more detail. It makes sense to me if we do what you said earlier actually and change these `Options` classes to just be constructor arguments instead, and the optional ones can just be positional args. Ill create a work item to track that work that will take place in a separate PR.